regions/core/bounding_box.py

```diff
@@ -153,14 +164,31 @@ def extent(self):
 
     def as_patch(self, **kwargs):
         """
-        Return a Matplotlib patch that represents the bounding box.
+        Return a `matplotlib.patches.Patch` that represents the bounding
+        box.
```

Comment:
Link to the more explicit `matplotlib.patches.Rectangle` here instead of `matplotlib.patches.Patch`?
Put this info in a `Returns` section?